sql-plugin/src/main/scala/org/apache/spark/rapids/hybrid/HybridExecutionUtils.scala

```diff
@@ -0,0 +1,262 @@
+      classOf[Flatten],
+      classOf[Floor],
+      classOf[FromUTCTimestamp],
+      classOf[FromUnixTime],
+      classOf[GetJsonObject],
+      classOf[GetMapValue],
+      classOf[GreaterThan],
+      classOf[GreaterThanOrEqual],
+      classOf[Greatest],
+      classOf[Hex],
+      classOf[Hour],
+      classOf[If],
+      classOf[In],
+      classOf[IsNaN],
+      classOf[IsNotNull],
+      classOf[IsNull],
+      classOf[LastDay],
+      classOf[Least],
+      classOf[Left],
+      classOf[Length],
+      classOf[LengthOfJsonArray],
+      classOf[LessThan],
+      classOf[Levenshtein],
+      classOf[Like],
+      classOf[Literal],
+      classOf[Log],
+      classOf[Log10],
+      classOf[Log2],
+      classOf[Lower],
+      classOf[MapFromArrays],
+      classOf[MapKeys],
+      classOf[MapValues],
+      classOf[MapZipWith],
+      classOf[Md5],
+      classOf[MicrosToTimestamp],
+      classOf[MillisToTimestamp],
+      classOf[Minute],
+      classOf[MonotonicallyIncreasingID],
+      classOf[Month],
+      classOf[NaNvl],
+      classOf[NextDay],
+      classOf[Not],
+      classOf[Or],
+      classOf[Overlay],
+      classOf[Pi],
+      classOf[Pow],
+      classOf[Quarter],
+      classOf[Rand],
+      classOf[Remainder],
+      classOf[Reverse],
+      classOf[Rint],
+      classOf[Round],
+      classOf[Second],
+      classOf[Sha1],
+      classOf[Sha2],
+      classOf[ShiftLeft],
+      classOf[ShiftRight],
+      classOf[Shuffle],
+      classOf[Sin],
+      classOf[Size],
+      classOf[SortArray],
+      classOf[SoundEx],
+      classOf[SparkPartitionID],
+      classOf[Sqrt],
+      classOf[Stack],
+      classOf[StringInstr],
+      classOf[StringLPad],
+      classOf[StringRPad],
+      classOf[StringRepeat],
+      classOf[StringReplace],
+      classOf[StringToMap],
+      classOf[StringTrim],
+      classOf[StringTrimLeft],
+      classOf[StringTrimRight],
+      classOf[Substring],
+      classOf[SubstringIndex],
+      classOf[Tan],
+      classOf[Tanh],
+      classOf[ToDegrees],
+      classOf[ToRadians],
+      classOf[ToUnixTimestamp],
+      classOf[UnaryPositive],
+      classOf[Unhex],
+      classOf[UnixMicros],
+      classOf[UnixMillis],
+      classOf[UnixSeconds],
+      classOf[Upper],
+      classOf[Uuid],
+      classOf[WeekDay],
+      classOf[WeekOfYear],
+      classOf[WidthBucket],
+      classOf[Year],
+      classOf[ZipWith]
+    )
+
+    // Some functions are fully supported with ANSI mode off
+    lazy val ansiOff = Seq(
+      classOf[Remainder],
+      classOf[Multiply],
+      classOf[Add],
+      classOf[Subtract],
+      classOf[Divide],
+      classOf[Abs],
+      classOf[Pmod]
+    )
+
+    if (SQLConf.get.ansiEnabled) {
+      ansiOn
+    } else {
+      ansiOn ++ ansiOff
+    }
+  }
+
+  def canBePushedToHybrid(child: SparkPlan, conf: RapidsConf): String = {
+    child match {
+      case fsse: FileSourceScanExec if HybridFileSourceScanExecMeta.useHybridScan(conf, fsse) =>
```

Comment:
Better to move `HybridFileSourceScanExecMeta.useHybridScan` to outer function.

Reply:
How about moving the function useHybridScan to HybridExecutionUtils?
And maybe all other functions in object HybridFileSourceScanExecMeta can be moved there too, what do you think?

Reply:
done